pkg/instrumentation/annotationmutator.go

```diff
@@ -0,0 +1,106 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package instrumentation
```

Comment:
wondering it files related to annotation should be in the annotation package instead of instrumentation ??

Reply:
This is all in service of instrumentation. Similar to how the `podmutator` is in the `instrumentation` package.